portfolio/src/main/webapp/index.html

```diff
@@ -174,8 +174,44 @@ <h4>
         <div class="info">     
           <h2>Projects</h2>
 
-          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-          </p>
+          <br>
+          <br>
+
+
+          <div class="container">
+            <div class="card">
+              <div class="imgBx">
+                <img src=images/motm.jpg width="200" height="175">
```

Comment:
Consider adding the width/height styling in the CSS instead.

Reply:
Ignore me :P, I realized Andrew had mentioned that adding in the HTML allows the browser to see how big it is before it downloads or parses CSS so it can avoid a second relayout after the image downloads.

Reply:
I think Andrew mentioned in an earlier PR to include width/height attributions on an image instead of CSS so the browser can better understand how much pixels need to be shifted? Should I still change this?